pkgs/development/python-modules/django-q2/default.nix

```diff
@@ -24,7 +24,7 @@ buildPythonPackage rec {
   src = fetchFromGitHub {
     owner = "django-q2";
     repo = "django-q2";
-    rev = "refs/tags/v${version}";
+    tag = "refs/tags/v${version}";
```

Comment:
You forgot to remove res/tags here

Reply:
My bad, probably missed because nix found a match for the hash and never tried to redownload the source.